Makefile

```diff
@@ -299,3 +312,9 @@ cutrelease: bumpversion generate-all-in-one helm-docs
 build-validator: ginkgo ## Build Ginkgo test binary.
 	CGO_ENABLED=0 $(GINKGO) build -tags=externaltests ./internal/hostnetwork
 	mv internal/hostnetwork/hostnetwork.test $(VALIDATOR_PATH)
+
+.PHONY: install-containerlab
+install-containerlab:
+	@if ! command -v clab >/dev/null 2>&1; then \
+	    curl -sL https://containerlab.dev/setup | sudo -E bash -s "install-containerlab"; \
```

Comment:
should we pin a clab version here?

Reply:
yep, this made me feel bad but that setup script runs either apt or dnf to deploy clab and I did not find a straightforward way to pin a version locally, nor I was able to make it work with podman in podman.
Let me try to spend a bit more trying to pursue this last option

Reply:
after sweating a bit I am giving up. I resorted to notify the user of the need to deploy clab vs deploying it ourselves, seems more respectful as we are deploying something on the host of the user.
The docker variant still runs with docker in docker, so in CI we won't have version skimming issues.